frontend/src/components/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import styles from './collapsible.module.css';
+import { up, down } from "./icons";
+
+type CollapsibleSection = {
+  title: string;
+  children: JSX.Element | JSX.Element[];
+}
+
+const Collapsible = ({ title, children }: CollapsibleSection) => {
+  const [expanded, setExpanded] = useState(false);
+  const icon = expanded ? down : up;
```

Comment:
I think these icons might need to be flipped? To hide the content I think you'd want to click an up button and to reveal content I think you'd want to click a down button. You could double check the designs or ask a designer, whichever works best for you.